content/influxdb/v2.2/tools/grafana.md

```diff
@@ -161,47 +183,115 @@ Provide the following:
+<!-------------------------- END OSS InfluxQL setup -------------------------->
+<!---------------------------------------------------------------------------->
+
+<!---------------------------------------------------------------------------->
+<!------------------------ BEGIN Cloud InfluxQL setup ------------------------>
+<!---------------------------------------------------------------------------->
+{{% cloud-only el="div" %}}
+
+To query InfluxDB Cloud from Grafana using InfluxQL:
+
+1. [Download and set up the `influx` CLI](#download-and-set-up-the-influx-cli)
+2. [Create an InfluxDB DBRP mapping](#create-an-influxdb-dbrp-mapping)
+3. [Configure your InfluxDB connection](#configure-your-influxdb-connection)
+
+### Download and set up the influx CLI
+1. [Download the latest version of the `influx` CLI](/influxdb/cloud/sign-up/#optional-download-install-and-use-the-influx-cli)
+   appropriate for your local operating system.
+2. Create a CLI configuration that provides the required InfluxDB Cloud **host**,
+   **organization**, and **API token** to all CLI commands.
+   Use the [`influx config create` command](/influxdb/cloud/reference/cli/influx/config/create/)
+   and provide the following:
+
+   - [InfluxDB Cloud URL](/influxdb/cloud/reference/regions/)
+   - [organization name](/influxdb/cloud/organizations/) _(by default, your email address)_
+   - [API token](/influxdb/cloud/security/tokens/)
+
+    ```sh
+    influx config create \
+      --config-name example-config-name \
+      --host-url https://cloud2.influxdata.com \
+      --org example-org \
+      --token My5uP3rSeCr37t0k3n
+    ```
+
+    For more information about `influx` CLI configurations,
+    see [`influx config`](/influxdb/cloud/reference/cli/influx/config/).
+
+### Create an InfluxDB DBRP mapping
+When using InfluxQL to query InfluxDB Cloud, the query must specify a database and a retention policy.
+Use the [`influx v1 dbrp create` command](/influxdb/cloud/reference/cli/influx/v1/dbrp/create/)
+command to create a database/retention policy (DBRP) mapping that associates a database
+and retention policy combination with an InfluxDB Cloud [bucket](/influxdb/cloud/reference/glossary/#bucket).
+
+DBRP mappings do not affect the retention period of the target bucket.
+These mappings allow queries following InfluxDB 1.x conventions to successfully
+query InfluxDB Cloud buckets.
+
+{{% note %}}
+##### Automatically create DBRP mappings on write
+When using the InfluxDB 1.x compatibility API to write data to InfluxDB Cloud,
+InfluxDB Cloud automatically creates DBRP mappings for buckets whose names match the
+`db/rp` naming pattern of the database and retention policy specified in the write request.
+For more information, see [Database and retention policy mapping – Writing data](/influxdb/cloud/reference/api/influxdb-1x/dbrp/#when-writing-data).
+{{% /note %}}
+
+Provide the following:
+
+- database name
+- retention policy name _(not retention period)_
```

Comment:
```suggestion
- [retention policy](/influxdb/v1.8/concepts/glossary/#retention-policy-rp) name _(not retention period)_
```

Reply:
would it be helpful to explicit state here (or elsewhere if we don't already cover):

To view 1.x [retention policies](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#retention-policy-rp), use the InfluxQL command [SHOW RETENTION POLICIES](https://docs.influxdata.com/influxdb/v1.8/query_language/explore-schema/#show-retention-policies).